### PR TITLE
Add Translation to TooltipDescription

### PR DESCRIPTION
**Pull request by dnqbob**

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/TooltipDescription.cs [hunk 1]
```diff
@@ -16,6 +16,7 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Additional info shown in the battlefield tooltip.")]
 	public class TooltipDescriptionInfo : ConditionalTraitInfo
 	{
+		[TranslationReference]
 		[Desc("Text shown in tooltip.")]
 		public readonly string Description = "";
```
Comment (PunkPun) [inline OpenRA.Mods.Common/Traits/TooltipDescription.cs]:
```suggestion
		public readonly string Description;
```

For better linting. This property is the whole point of the trait so it should be required

File: mods/cnc/rules/tech.yaml [hunk 2]
```diff
@@ -13,10 +13,10 @@ V19:
 	Tooltip:
 		Name: actor-v19-name
 	TooltipDescription@ally:
-		Description: Provides additional funds.
+		Description: actor-oilb-desc1
```
Comment (PunkPun) [inline mods/cnc/rules/tech.yaml]:
```suggestion
		Description: actor-oilb.ally-desc
```

It would be better if translations were structure like this

**Review by PunkPun — COMMENTED** [review]
Otherwise lgtm

File: mods/cnc/rules/tech.yaml (hunk 2 above)
Comment (dnqbob) [reply mods/cnc/rules/tech.yaml]: fixed

File: OpenRA.Mods.Common/Traits/TooltipDescription.cs (hunk 1 above)
Comment (dnqbob) [reply OpenRA.Mods.Common/Traits/TooltipDescription.cs]: fixed

**Review by PunkPun — APPROVED** [review]
LGTM

**Comment by PunkPun** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/08feacb89673ac07ac9bb893dd4f5d8b2ac8bc61...cf462b585bba2ac234a7e8483aa1e5b8c9845d5b)

**Pull request merged**